src/libs/Network.js

```diff
@@ -68,7 +68,8 @@ function processNetworkRequestQueue() {
         }
 
         const requestData = queuedRequest.data;
-        if (email) {
+        const requestEmail = requestData.email ?? '';
+        if (email && _.isEmpty(requestEmail)) {
```

Comment:
I'm assuming this change was made so that we would not overwrite the value when setting a secondary login. Does this mean those logs are weird and show the secondary login as the actor in the logs? Maybe could add a comment here so that someone doesn't try to break things.

Reply:
Basically what was happening was that for `SecondaryLogin_Send` we were passing an `email` and a `password` in the `requestData` and it was getting overwritten here with the email associated with the person logged in. 

This resulted in us basically trying to add the user's primary login as a new secondary login. So I modified this so we only overwrite the value if there isn't already an `email` being passed.

Reply:
I can add a comment to clarify

Reply:
Cool. Thanks, but it's still weird right? We add the email so that the logs show as coming from the correct person. Maybe it's the same everywhere. 🤷‍♂️